redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/DatabaseV4Endpoint.java

```diff
@@ -81,12 +97,10 @@ DatabaseV4Response delete(
     @DELETE
     @Path("")
     @Consumes(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = DatabaseOpDescription.DELETE_MULTIPLE_BY_NAME, notes = DatabaseNotes.DELETE_MULTIPLE_BY_NAME,
-            nickname = "deleteMultipleDatabases")
+    @ApiOperation(value = DatabaseOpDescription.DELETE_MULTIPLE_BY_CRN, notes = DatabaseNotes.DELETE_MULTIPLE_BY_CRN,
+            nickname = "deleteMultipleDatabasesByCrn")
     DatabaseV4Responses deleteMultiple(
```

Comment:
Should this be renamed to deleteMultipleByCrn?

Reply:
Unnecessary, because ByCrn or ByName are only there if there is a choice of endpoints. Here, there's only one deleteMultiple.